source/features/list-prs-for-file.tsx

```diff
@@ -31,17 +34,14 @@ function getDropdown(prs: number[]): HTMLElement {
 					File touched by PRs
 				</div>
 				{prs.map(prNumber => (
-					<li
```

Comment:
If you drop the `li`, then it's no longer valid because there's an `ul` around it. It must be fixed either way